website/docs/docs/on-premises/releases/1.29.0.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 1.29.0
+---
+
+## Release date
+
+30 January 2024
+
+## Features
+
+* The GET /integrations endpoint returns verification modes: replay or comparison along with other integration details
+* Added OAS for consumer contract publication
```

Comment:
This was a change for the OAS, not the Pactflow product, so it can be removed from these release notes.